README.md

```diff
@@ -49,6 +49,8 @@ Versions:
 
 SkroutzGreekStemmer Plugin | ElasticSearch
 ---------------------------|------------------
+0.0.13                     | 2.3.1
+0.0.12                     | 1.7.x
```

Comment:
You are right. I thought it was published but was not updated in the README.
I will remove both of those lines.